assets/multiqc_config.yaml

```diff
@@ -0,0 +1,9 @@
+report_comment: >
```

Comment:
`multiqc_config.yaml` is in both `conf` and `assets`. Keep the one in assets only.

Reply:
done